src/v/cluster/cloud_metadata/manifest_downloads.cc

```diff
@@ -102,7 +102,7 @@ ss::future<cluster_manifest_result> download_highest_manifest_for_cluster(
     }
 
     // Deserialize the manifest.
-    auto manifest_res = co_await remote.download_manifest(
+    auto manifest_res = co_await remote.download_manifest_json(
```

Comment:
so there were no callers to download manifest with the serde type? i'd have expected at least one rename to _bin

Reply:
That's right, download_manifest's definition was json only, and these usages of it are explicitly manifest types that we've only ever written in json